extensions/interactions/CodeRepl/CodeRepl.js

```diff
@@ -119,7 +119,7 @@ oppia.directive('oppiaInteractiveCodeRepl', [
           execLimit: 10000
         });
 
-        $scope.runCode = function(codeInput) {
+        $scope.runAndSubmitCode = function(codeInput, status) {
```

Comment:
umm.. Isn't second argument required to tell the function which button has been clicked ?
I just tried to adjust all in one function because second one was almost the same except that one line missing .
I could do it that way .. But that still requires an extra arg ??